docs/running-local.md

```diff
@@ -0,0 +1,236 @@
+
+For example on Ubuntu 16.04 (LTS):
+
+    sudo apt-get install -y docker docker-compose
+    sudo adduser $USER docker
+    sudo su $USER # so that the group is taken into account
+    docker ps # should not complain :)
+
+Get Secotrec-local
+------------------
+
+Then we get the `secotrec-local` application:
+
+    mkdir ~/bin/
+    wget -O ~/bin/secotrec-local https://storage.googleapis.com/smondet-experiments/secotrec-local-Linux-x86_64-03
+    chmod +x ~/bin/secotrec-local
+    export PATH=~/bin:$PATH
+
+And do a quick test:
+
+    secotrec-local --help
+
+Local Infrastructure Deployment
+-------------------------------
+
+Secotrec-local is configured with environment variables, you can put them in a
+file and source it, or just paste them in your shell:
+
+    # HTTPS port to access the Ketrew UI:
+    export tls_port=443
+    # Directory where Epidisco will write files
+    export biokepi_work=$HOME/biokepi-work/
+    # Long, random, secret string of you choice:
+    export ketrew_auth_token=dsleaijdej308098ddecja9c8jra8cjrf98r
+    # The maximal number of jobs that are allowed to run at once:
+    export coclobas_max_jobs=8
+    # We also want a configured Epidisco development environment:
+    export epidisco_dev=$HOME/epidisco-shared
+    # We want to get recent version of most tools:
+    export coclobas_docker_image=hammerlab/keredofi:coclobas-aws-biokepi-dev
+
+You check that the configuration is taken into account:
+
+    secotrec-local print-conf
+
+You need this set up (the first time, for now):
+
+    mkdir -p $epidisco_dev
+    chmod -R 777 $epidisco_dev
+    mkdir -p /tmp/secotrec-local-shared-temp
+    chmod -R 777 /tmp/secotrec-local-shared-temp
+
+Then you can launch the setup:
+
+    secotrec-local up
+
+the first time it takes a while because it pulls docker images.
+
+Check the status of the deployment with:
+
+    secotrec-local status
+
+Using the hostname or IP address of your host, you can access the Ketrew WebUI
+at (using the auth-token you defined above):
+
+`https:// ... /gui?token=dsleaijdej308098ddecja9c8jra8cjrf98r`
+
+
+Submit a First Epidisco Run
+---------------------------
+
+Let's generate the Biokepi “infrastructure” configuration:
+
+    secotrec-local biokepi-machine $epidisco_dev/biokepi_machine.ml
+
+and then *enter* the docker-container which has all the necessary dependencies
+(incl. Epidisco itself):
+
+    secotrec-local docker-compose -- exec epidisco-dev opam config exec bash
+
+Let's configure the client-side for Ketrew (fill your host like above):
+
+    ketrew init --just-client http://kserver:8080/gui?token=dsleaijdej308098ddecja9c8jra8cjrf98r
+
+Let's just run a quick test of the configuration:
+
+    ketrew submit --dae /tmp/,"du -sh /epidisco-shared" --wet-run
+
+The job should be visible on the Ketrew WebUI (you can also use the “TextUI” by
+simply typing `ketrew interact` and following the 80's-styled menus).
+
+We just need to get the “epidisco runner:”
```

Comment:
It should be equivalent to the `run_pipeline.ml` script and I think it might be outdated already. Again, this part will/should be much smoother in the final doc.